### ISSUE TITLE
Possible Perm Ban?

### ISSUE THREAD

**Issue by mcfowlz** [jabbink/PokemonGoBot#1025]
I'm getting this message:

12 Aug 11:35:40 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:41 [default: PokestopLoop] - Pokestop out of range; distance: 36.018527301210405
12 Aug 11:35:41 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:41 [default: BotLoop] - Found pokemon WEEPINBELL
12 Aug 11:35:42 [default: PokestopLoop] - Pokestop out of range; distance: 36.018527301210405
12 Aug 11:35:42 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:42 [default: BotLoop] - Encountered pokemon WEEPINBELL with CP 217 and IV 62%
12 Aug 11:35:42 [default: BotLoop] - Using ITEM_ULTRA_BALL; Using curve; achieved catch probability: 0.57036, desired: 0.4
12 Aug 11:35:43 [default: BotLoop] - Capture of WEEPINBELL failed with status : CATCH_FLEE
12 Aug 11:35:43 [default: PokestopLoop] - Pokestop out of range; distance: 36.018527301210405
12 Aug 11:35:43 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:44 [default: PokestopLoop] - Pokestop out of range; distance: 36.018527301210405
12 Aug 11:35:44 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:44 [default: PokestopLoop] - Pokestop out of range; distance: 36.018527301210405
12 Aug 11:35:44 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:45 [default: BotLoop] - Getting map of (40.78404551739303, -73.96809766214682)
12 Aug 11:35:45 [default: PokestopLoop] - Pokestop out of range; distance: 40.657362808664054
12 Aug 11:35:45 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:45 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:45 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:46 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:46 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:46 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:46 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:47 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:47 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:48 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:48 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:49 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:49 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:49 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:49 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:50 [default: BotLoop] - Getting map of (40.78409298777123, -73.96816393898334)
12 Aug 11:35:50 [default: PokestopLoop] - Pokestop out of range; distance: 33.0110188764431
12 Aug 11:35:50 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:50 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:50 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:51 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:51 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:52 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:52 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:52 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:52 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:53 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:53 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:53 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615
12 Aug 11:35:53 [default: PokestopLoop] - Looting nearby pokestop 2e1932a7813a419eadb70ce0faf992d0.16
12 Aug 11:35:54 [default: PokestopLoop] - Pokestop out of range; distance: 34.531190011048615

Can anyone explain?

**Comment by mcfowlz** [comment]
also i went on my actual Pgo app, i can get onto it but can't do gym battles and when i try to spin pokestops it says 'try again later'

**Comment by aukeboskma** [comment]
Same here but I can do gyms.

**Comment by staurakis** [comment]
Check [this issue](https://github.com/jabbink/PokemonGoBot/issues/1023)

**Comment by doness** [comment]
you got softban

**Comment by kuklaxklan** [comment]
how are you.

I got this error message. Its not a soft ban but it is a Ban but can figure out how to by pass it. Spinning the Poke stop doesnt help. I went and opend itwith my phone when i spin usually it wouldnt say anything and it wouldnt give anything back but now it say "try again later"  its been 6 hours still says try again later. some pokemons are running away some i am able to catch and find but i am not able to get Anything from Pokestops.

Pleas help. Here is the log

[05:58:55] Name: Songwriters Hall of Fame INFO: Looting failed, possible softban. Unban in: 45/45
[05:58:57] Traveling to Pokestop: Gothic Archway at Providence (107m) (4 seconds)
[05:58:57] Looking for Pokemon...
[05:59:00] (CatchEscape) | (Normal) Weedle Lvl: 14 CP: (143/179) IV: 38.00% | Chance: 40.01% | 49.76m dist | with a PokeBall (191 left). | lat: 40.767414 long: -73.983049
[05:59:02] (CatchSuccess Attempt #2) | (Normal) Weedle Lvl: 14 CP: (143/179) IV: 38.00% | Chance: 40.01% | 49.76m dist | with a PokeBall (190 left). | 210 EXP earned | Candies: 7 | lat: 40.767414 long: -73.983049
[05:59:03] (CatchEscape) | (Normal) Weedle Lvl: 22 CP: (200/282) IV: 11.00% | Chance: 31.92% | 57.25m dist | with a PokeBall (189 left). | lat: 40.766630 long: -73.983508
[05:59:05] (CatchSuccess Attempt #2) | (Normal) Weedle Lvl: 22 CP: (200/282) IV: 11.00% | Chance: 31.92% | 57.25m dist | with a PokeBall (188 left). | 210 EXP earned | Candies: 10 | lat: 40.766630 long: -73.983508
[05:59:06] Looking for incense Pokemon...
[05:59:06] Looking for Pokemon...
[05:59:07] Looking for incense Pokemon...
[05:59:11] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 1/45
[05:59:12] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 2/45
[05:59:13] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 3/45
[05:59:14] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 4/45
[05:59:16] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 5/45
[05:59:17] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 6/45
[05:59:18] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 7/45
[05:59:19] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 8/45
[05:59:21] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 9/45
[05:59:22] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 10/45
[05:59:24] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 11/45
[05:59:25] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 12/45
[05:59:26] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 13/45
[05:59:28] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 14/45
[05:59:29] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 15/45
[05:59:30] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 16/45
[05:59:32] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 17/45
[05:59:33] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 18/45
[05:59:34] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 19/45
[05:59:36] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 20/45
[05:59:37] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 21/45
[05:59:38] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 22/45
[05:59:40] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 23/45
[05:59:41] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 24/45
[05:59:42] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 25/45
[05:59:44] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 26/45
[05:59:45] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 27/45
[05:59:47] Name: Gothic Archway at Providence INFO: Looting failed, possible softban. Unban in: 28/45
